src/routes/(console)/onboarding/create-organization/+page.svelte

```diff
@@ -47,6 +47,10 @@
             if (organization) {
                 loadAvailableRegions(organization?.$id).then();
                 await goto(`${base}/organization-${organization.$id}`);
+
+                // fixes an edge case where
+                // the org is not available for some reason!
+                await invalidate(Dependencies.CREATE_ORGANIZATION);
```

Comment:
this is because we use `const { organizations } = await parent()`, which is loaded at the top level `+layout.ts`. Children below it don't load it so the first project creation can fail sometimes for billing checks because there's no org in the list. This just invalidates and loads all of them again. Perf cost is very minimal.